dash_app/dashboard.py

```diff
@@ -196,6 +258,7 @@ def init(server):
     )
 
     _init_callbacks(dash_app, ds, per_timeseries_stats, region_df["id"])
+    print(show_callbacks(dash_app))
```

Comment:
Intentionally leaving this in?  (seems potentially useful, just double-checking)